### PR TITLE
Add some constructor arguments to some layouts

### PR DESCRIPTION
**Pull request by psychon**
This adds new constructor arguments to align, fixed and flex which allows adding
widgets directly while creating the layout.

Idea originally by actionless:
https://github.com/awesomeWM/awesome/pull/486#issuecomment-143606301

@actionless What do you think? :-)

### CODE REVIEW

**Comment by actionless** [comment]
cool! that's what i wanted

File: lib/wibox/layout/fixed.lua [hunk 1]
```diff
@@ -49,10 +49,15 @@ function fixed:layout(context, width, height)
     return result
 end
 
---- Add a widget to the given fixed layout
-function fixed:add(widget)
-    base.check_widget(widget)
-    table.insert(self.widgets, widget)
+--- Add some widgets to the given fixed layout
+function fixed:add(...)
+    -- No table.pack in Lua 5.1 :-(
```
Comment (Elv13) [inline lib/wibox/layout/fixed.lua]:
````lua
function foo(...) local packed = {...}; print(#packed); end
foo(1,2,3,4)
>> 4
````

Seem to work for me in 5.1

**Comment by blueyed** [comment]
Great.

Would be nice to have documentation for this.

**Comment by psychon** [comment]
Done. Feel free to `--autosquash`. :-)

**Comment by blueyed** [comment]
Thanks, merged!

**Pull request closed without merging**